interface/forms/CAMOS/notegen.php

```diff
@@ -50,13 +51,7 @@
     <?php echo xlt('Print Notes'); ?>
 </title>
 
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker/build/jquery.datetimepicker.min.css">
-
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery/dist/jquery.min.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker/build/jquery.datetimepicker.full.min.js"></script>
-<script type="text/javascript" src="<?php echo $depth ?>library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="<?php echo $depth ?>library/textformat.js?v=<?php echo $v_js_includes; ?>"></script>
+    <?php Header::setupHeader('datetime-picker'); ?>
```

Comment:
What happened to dialog.js and textformat.js here?

Reply:
Those are autoloaded in the setupHeader call. The following are autoloaded:
jquery
main-theme
bootstrap
fontawesome
utility.js
textformat.js
dialog.js